app/__init__.py

```diff
@@ -0,0 +1,16 @@
+from flask_restplus import Api
+from flask import Blueprint
+
+from .main.controller.user_controller import api as user_ns
+from .main.controller.auth_controller import api as auth_ns
+
+blueprint = Blueprint('api', __name__)
+
+api = Api(blueprint,
+          title='FLASK RESTPLUS API BOILER-PLATE WITH JWT',
+          version='1.0',
+          description='a boilerplate for flask restplus web service'
+          )
+
+api.add_namespace(user_ns, path='/user')
```

Comment:
Add description to both the name spaces.

Reply:
I think It'll be better to remove these namespaces for now since our paths aren't defined. Once we add the endpoints, we can add the namespaces

Reply:
Agreed.

Reply:
I'm removing the second namespace since as of now, we haven't defined any endpoint or paths. It'll be easier to add it once we add an authentication endpoint.